crates/executor/src/executor/mod.rs

```diff
@@ -205,7 +205,7 @@ where
             cumulative_gas_used += result.gas_used();
 
             // Create receipt envelope.
-            let receipt = receipt_envelope_from_parts(
+            let receipt = OpReceiptEnvelope::<Log>::from_parts(
```

Comment:
This is a bit dangerous, as upstream alloy allows 7702 txs through as Ithica needs it for Odyssey, but we explicitly need to disallow them. We need to make sure that we add enforcement for fork activation.